corehq/apps/ota/tests/test_search_claim_endpoints.py

```diff
@@ -141,6 +144,43 @@ def test_claim_restore_as(self):
         claim_case = CaseAccessors(DOMAIN).get_case(claim_ids[0])
         self.assertEqual(claim_case.owner_id, other_user._id)
 
+    def test_claim_restore_as_proper_cache(self):
```

Comment:
yeah i thought about it, but decided i wasn't testing that functionality with this test and didn't want to add the overhead